Lib/logging/handlers.py

```diff
@@ -1093,8 +1095,19 @@ def emit(self, record):
             msg.set_content(self.format(record))
             if self.username:
                 if self.secure is not None:
+                    try:
+                        keyfile = self.secure[0]
+                    except IndexError:
+                        keyfile = None
```

Comment:
A newline for readability would be nice
```suggestion
                        keyfile = None

```

Reply:
Done